### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by girma336**
-Added  list of item of the home page

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -52,13 +53,100 @@ nav a:hover {
...
+  border-radius: 12px;
+  transition: 0.5s;
+}
+
+.comment-rese button:hover {
+  background: blue;
+  color: #fff;
 }
 
 footer {
```
Comment (Victoria129) [inline src/css/style.css:152]: Kindly remove empty elements for best practice.

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
```
Comment (Victoria129) [inline src/index.html:7]: Kindly change title to reflect the project.

File: src/css/style.css (hunk 1 above)
Comment (girma336) [reply src/css/style.css:152]: - Ok but not empty because it has comment and reservation button.

File: src/index.html (hunk 2 above)
Comment (girma336) [reply src/index.html:7]: - ok thank you i will do it now

**Review by Victoria129 — CHANGES_REQUESTED** [review]
Hi @girma336,
Good job so far!
Please refer to the inline comments for the required changes.

File: src/css/style.css (hunk 1 above)
Comment (Victoria129) [reply src/css/style.css:152]: Oh...okay. If it is not empty, then you can ignore the comment.

**Review by Victoria129 — APPROVED** [review]
Hi @girma336 🤝 ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Highlights
👍🏽 Your project looks great, I like the effort you put in in order to complete it, keep up the great work 💪🏽
👍🏽 The PR follows the correct git / github flow ✔️

**Pull request merged**